Automattic/jetpack
# Fixes the production build script.

#### Changes proposed in this Pull Request:
- Makes sure that `npm run build-production` works well even without previous npm installations.
- Changes the way we use i18n: actually using `i18n-calypso` now, giving us the opportunity to add the textdomain string.
- Gathering translatable strings from uncompiled JavaScript code to have references to files and line numbers.

#### Testing instructions:
- Run `npm run build-script`, note that it would take a lot of time and generate lots of new language files. The main thing that we're currently interested in is the `_inc/jetpack-strings.php` file, it should contain all the translatable strings from `_inc/client/**/*.js*` files.
